test/controllers/user.test.js

```diff
@@ -66,3 +69,35 @@ describe('usersController.signUp', () => {
       });
   });
 });
+
+describe('usersController.screateUserignIn', () => {
+  beforeEach(() => {
+    userAttributes.email = 'omar.rodriguez@wolox.com';
+    userAttributes.password = 'password1923';
+    return createUser();
```

Comment:
You should not pass params to a function by editing an existing object. This is super hard to follow for other people looking at your code.
`createUser` should either receive both email and pass or an object containing them.